src/board/mod.rs

```diff
@@ -579,32 +577,16 @@ impl Board {
             .count()
     }
     
-    pub fn new_chain_liberties(&self, m: Move) -> usize {
```

Comment:
Yes, exactly. Maybe it's possible to move those methods as helper functions into the test mod. Then they're out of the `Board` struct but the tests remain more or less the same.